api/labels_medium_test.go

```diff
@@ -0,0 +1,74 @@
+// +build medium
+
+// Copyright 2019 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package api
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+	"github.com/web-platform-tests/wpt.fyi/shared/sharedtest"
+	"google.golang.org/appengine/datastore"
+)
+
+func TestLabelsHandler(t *testing.T) {
+	ctx, done, err := sharedtest.NewAEContext(true)
+	assert.Nil(t, err)
+	defer done()
+
+	key := datastore.NewIncompleteKey(ctx, "TestRun", nil)
+	_, err = datastore.Put(ctx, key, &shared.TestRun{Labels: []string{"a"}})
```

Comment:
Can you add the labels out of order to also test the sorting, and perhaps also a duplicate?

Reply:
Done.